Cheetahs - Lynn Jansheski

1lynnj opened the pull request:
# Personal Portfolio Site
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
:------------- | :-------------
Did you have to resolve any issues when running the HTML Validator? If so, what were they? |The only issues I had with the validator is if I left off a closing tag or my tags didn't match up.
Why is it important to consider and use semantic HTML? |The semantic tags helped me keep track of what parts of the html I was styling in my css.
How did you decide to structure your CSS? |One file. The top styles are site wide then grouped by section of the page. 
What was the most challenging piece of this assignment? |Using css grid. Flexbox either has better documentation or it was more intuitive. I'm not sure why but I just had a lot easier time using flexbox. Another challenging piece was writing DRY code with css. In trying to understand how css works, I used a lot of trial and error and ended up with a lot of extraneous code. I cleaned it up by commenting out css code line by line to see what was actually affecting the styling. I deleted a lot more lines that I was expecting.
Describe one area that you gained more clarity on when completing this assignment |How the css works. It's still a skill in its infancy, but I have a much greater understanding having practiced.
**Optional** Did you deploy to GitHub Pages? If so, what is the URL to your website? |lynnjansheski.com

1lynnj committed:
Created initial file structure and styles sheet. Updated head in index.

1lynnj committed:
Added images to images folder and added nav links to header.

1lynnj committed:
Added main to index. Includes sidebar.

1lynnj committed:
Added footer content including image.

1lynnj committed:
Started grid layout for index

1lynnj committed:
Working on grid

1lynnj committed:
Completed initial layout of index page

1lynnj committed:
Updating styles for index

1lynnj committed:
Added links for about, portfolio and contact pages to index. Added links to side bar.

1lynnj committed:
Added content to about page. Created about.css file, started basic styling of about page.

1lynnj committed:
Added initial content and created stylesheet for portfolio page.

1lynnj committed:
Playing with styling. Need to clean up all of the unused code.

1lynnj committed:
Finalized styling for home page

1lynnj committed:
Updated about page and added about css file. Added specific styling to about.css

1lynnj committed:
Updated portfolio page. Added portfolio css file and specific styling to portofolio.css

1lynnj committed:
Removed commented out code

1lynnj committed:
Working on tweaking styling.

1lynnj committed:
Added content and styling to portfolio page. Started cleaning up and reorganizing html and corresponding css.

1lynnj committed:
Cleaning up styles - compiled all into one, cleaned up labels, fixed styling errors, created styling for narrow devices.

1lynnj committed:
added targets to all of the contact links

1lynnj committed:
Reorganized directories to work with web hosting site

1lynnj committed:
Missed a directory change

1lynnj committed:
Removed target from portfolio link in about me

jbieniosek commented on index.html:
```diff
@@ -0,0 +1,51 @@
+  <link rel="stylesheet" href="/stylesheets/fonts.css">
+</head>
+
+  <body class="home">
+
+    <main class="page_grid">
```
This main tag here is wrapping all of the content - I recommend moving the class from this tag into the body tag, and replacing the section tag below with main:
```suggestion
  <body class="home page_grid">
```

jbieniosek commented on index.html:
```diff
@@ -0,0 +1,51 @@
+        <p>© Lynn Jansheski 2022</p>
+      </footer>
+
+    </main>
+
+  </body>
```
```suggestion

  </body>
```

jbieniosek commented on index.html:
```diff
@@ -0,0 +1,51 @@
+        </nav>
+      </header>
+
+      <section class="middle_row">
+        <div class="middle_content">
+          <aside class="side_bar">
+            <img class="profile_image" src="/images/profile_pic.png" alt="Profile Picture">
+            <p>Lynn Jansheski</p>
+            <address><a href="https://www.google.com/maps/place/Seattle,+WA/@47.6131746,-122.4821491,11z/data=!3m1!4b1!4m5!3m4!1s0x5490102c93e83355:0x102565466944d59a!8m2!3d47.6062095!4d-122.3320708" target="_blank" rel="noopener noreferrer"><img src="/images/location.png" alt="location icon">Seattle, WA</a></address>
+          </aside>
+          <div class="content">
+            <h1>Hi! I'm Lynn.</h1>
+            <h2>I'm a full stack <br>software engineer.</h2>
+          </div>
+        </div>
+      </section>
```
```suggestion
      <main class="middle_row">
        <div class="middle_content">
          <aside class="side_bar">
            <img class="profile_image" src="/images/profile_pic.png" alt="Profile Picture">
            <p>Lynn Jansheski</p>
            <address><a href="https://www.google.com/maps/place/Seattle,+WA/@47.6131746,-122.4821491,11z/data=!3m1!4b1!4m5!3m4!1s0x5490102c93e83355:0x102565466944d59a!8m2!3d47.6062095!4d-122.3320708" target="_blank" rel="noopener noreferrer"><img src="/images/location.png" alt="location icon">Seattle, WA</a></address>
          </aside>
          <div class="content">
            <h1>Hi! I'm Lynn.</h1>
            <h2>I'm a full stack <br>software engineer.</h2>
          </div>
        </div>
      </main>
```

jbieniosek commented on pages/about.html:
```diff
@@ -4,9 +4,60 @@
+
+    <main class="page_grid">
+
+      <header class="header">
+        <nav class="menu_bar">
```
👍

jbieniosek commented on stylesheets/styles.css:
```diff
@@ -0,0 +1,200 @@
+    background-color: #EAF4F4;
+}
+
+.menu_bar ul {
```
Nice flex layout!

jbieniosek commented on stylesheets/styles.css:
```diff
@@ -0,0 +1,200 @@
+}
+
+/* Page layout */
+.page_grid {
```
👍

jbieniosek reviewed (commented):
Great work on this project Lynn!  Nice use of semantic HTML.  This project is green.

1lynnj committed:
Updating content

1lynnj committed:
Updating content

1lynnj committed:
Added chatlog to my portfolio page

1lynnj committed:
adding bootstrap, moving about page info to index

1lynnj committed:
Refactoring to add bootstrap and add content

*removed grid classes
*updated links and added icons
*added bootstrap classes and updated custom styles

1lynnj committed:
Merge pull request #1 from 1lynnj/refactor

Refactoring to add bootstrap and add content

1lynnj committed:
fixed responsive styling

1lynnj committed:
Merge pull request #2 from 1lynnj/refactor

fixed responsive styling

1lynnj committed:
updating to present status

1lynnj committed:
Merge pull request #3 from 1lynnj/updates

Updating status
